code/modules/clothing/suits/marine_armor.dm

```diff
@@ -282,6 +282,77 @@
 	name = "\improper XM4 pattern intelligence officer armor"
 	uniform_restricted = list(/obj/item/clothing/under/marine/officer, /obj/item/clothing/under/rank/qm_suit, /obj/item/clothing/under/marine/officer/intel)
 	specialty = "XM4 pattern intel"
+	// XM4 Integral Motion Detector
+	actions_types = list(/datum/action/item_action/toggle, /datum/action/item_action/intel/toggle_motion_detector)
+	var/motion_detector = 0
```

Comment:
Code doc this as one can't tell what it is at a glace. Also if its supposed to be boolean then use `FALSE`
```
/// This var does something
var/my_var = 0
```

Reply:
How do I make a code doc? Or do you mean just add more comments?

Reply:
notice how review above has /// instead of //, this is code doc. makes things show up when hovering over it in vsc iirc

Reply:
Oh, I see what you mean. That’s neat, thanks!